.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,37 @@
+## Notes for Reviewer
+<!-- Items in addition to the checklist below that the reviewer should look for -->
+
+## Pre-Review Checklist for the PR Author
+
+1. [ ] Add sensible notes for the reviewer
+1. [ ] PR title is short, expressive and meaningful
+1. [ ] Relevant issues are linked
+1. [ ] Every source code file has the SPDX header `SPDX-License-Identifier: $(LICENSE_CODE)`
+1. [ ] Branch follows the naming format (`elk-123-introduce-posix-ipc-example`)
+1. [ ] Commits messages are according to this [guideline][commit-guidelines]
+    - [ ] Commit messages have the issue ID (`[#123] add posix ipc example`)
+    - [ ] Commit author matches [Eclipse Contributor Agreement][eca] (and ECA is signed)
+1. [ ] Tests follow the [best practice for testing][testing]
+1. [ ] Changelog updated [in the unreleased section][changelog] including API breaking changes
+1. [ ] Assign PR to reviewer
+1. [ ] All checks have passed (except `task-list-completed`)
+
+[commit-guidelines]: https://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html
+[eca]: http://www.eclipse.org/legal/ECA.php
+[testing]: https://github.com/eclipse-iceoryx/iceoryx/blob/master/doc/website/concepts/best-practice-for-testing.md
+[changelog]: https://github.com/larry-robotics/elkodon/blob/main/doc/release-notes/elkodon-unreleased.md
+
+## Checklist for the PR Reviewer
+
+- [ ] Commits are properly organized and messages are according to the guideline
+- [ ] Unit tests have been written for new behavior
+- [ ] Public API is documented
+- [ ] PR title describes the changes
+
+## Post-review Checklist for the PR Author
+
+1. [ ] All open points are addressed and tracked via issues
+
+## References
+
+- Closes **TBD**
```

Comment:
In iceoryx people quite often forget about this or have not idea what is meant. Could you please add a comment like `<!-- Use 'Closes #123' or 'Relates to #123' -->`